dao/src/main/resources/db/changelog/logs/ch-insert-into-tag-translations-Mokhnatska.xml

```diff
@@ -0,0 +1,99 @@
+            <column name="tag_id">21</column>
+            <column name="language_id">1</column>
+        </insert>
+        <insert tableName="tag_translations">
+            <column name="id">62</column>
+            <column name="name">Resource saving</column>
+            <column name="tag_id">21</column>
+            <column name="language_id">2</column>
+        </insert>
+        <insert tableName="tag_translations">
+            <column name="id">63</column>
+            <column name="name">Экономия ресурсов</column>
+            <column name="tag_id">21</column>
+            <column name="language_id">3</column>
+        </insert>
+        <insert tableName="tag_translations">
+            <column name="id">64</column>
+            <column name="name">Розумне споживання </column>
+            <column name="tag_id">22</column>
+            <column name="language_id">1</column>
+        </insert>
+        <insert tableName="tag_translations">
+            <column name="id">65</column>
+            <column name="name">Smart consuming</column>
+            <column name="tag_id">22</column>
+            <column name="language_id">2</column>
+        </insert>
+        <insert tableName="tag_translations">
+            <column name="id">66</column>
+            <column name="name">Умное потребление</column>
+            <column name="tag_id">22</column>
+            <column name="language_id">3</column>
+        </insert>
+        <insert tableName="tag_translations">
+            <column name="id">67</column>
+            <column name="name">Транспорт</column>
+            <column name="tag_id">23</column>
+            <column name="language_id">1</column>
+        </insert>
+        <insert tableName="tag_translations">
+            <column name="id">68</column>
+            <column name="name">Transportation</column>
+            <column name="tag_id">23</column>
+            <column name="language_id">2</column>
+        </insert>
+        <insert tableName="tag_translations">
+            <column name="id">69</column>
+            <column name="name">Транспорт</column>
+            <column name="tag_id">23</column>
+            <column name="language_id">3</column>
+        </insert>
+        <insert tableName="tag_translations">
+            <column name="id">70</column>
+            <column name="name">Переробити/сортувати відходи</column>
+            <column name="tag_id">24</column>
+            <column name="language_id">1</column>
+        </insert>
+        <insert tableName="tag_translations">
+            <column name="id">71</column>
+            <column name="name">Recycling / Waste sorting</column>
+            <column name="tag_id">24</column>
+            <column name="language_id">2</column>
+        </insert>
+        <insert tableName="tag_translations">
+            <column name="id">72</column>
+            <column name="name">Переделать/сортировать отходы</column>
```

Comment:
do we really need russian translation?

Reply:
I do not see the russian  version of the site.
![image](https://user-images.githubusercontent.com/15122465/214323182-5c8dd123-d944-488c-9b17-6bed67d82acd.png)

Reply:
we should not support russian
#russiaisaterroriststate

Reply:
I'll remove it